news/_posts/2020-09-10-bug-bounty-retrospective.md

```diff
@@ -0,0 +1,71 @@
+---
+layout: post
+title:  "Bug Bounty Program, the retrospective"
+subtitle: "When hunters make your business safer!"
+date:   2020-09-10 09:00:00
+authors: [ PierreRambaud, PrestaShop ]
+icon: icon-chat
+image: /assets/images/2020/09/ywhbanner.jpg
+tags: [bug, bounty, program, maintenance]
+---
+
+For several months, we experimented a Bug Bounty Program with the [YesWeHack](https://yeswehack.com) platform. Here is a retrospective, explaining why we did it, and what happened during this period.
+
+
+## Private program
+
+The security of software cannot be assessed unless it is tested by a lot of hunters.
+
+PrestaShop relies on a lot of dependencies, with hundred of thousands lines of code, and this has been true since the very first versions. 
+
+We considered that publishing a public program, available for all, in the first place was risky as it could bring a huge volume of duplicated reports and create frustration among hunters. That is why we chose a private program first.
+
+The benefit of the private program is to limit the number of hunters and manage the reporting flow. For each reported security issue, resources must be dedicated to understand it, reproduce it, build a patch, test the patch by the QA Team, etc. This can be quite expensive, so it is important to be able to adjust how many issues are reported, thanks to the private program settings.
+
+The process can appear quite slow and boring but it is nevertheless necessary to leave no room for doubt.
+
+
+## Public program
+
+After a few months running the private program, we noticed the number of reports was decreased significantly. We then decided to open the program to public the 23th of July 2020.
+
+We expected that the public opening would bring in a huge volume of new reports, but this did not happen to our great surprise. It seems the hunters who participated to the private program helped us fix a large majority of security issues.
+
+At the time of writing this article, only one report submitted after the opening of the program as public has been validated.
+
+The big aim of having a Bug Bounty Program in public is to show to everyone, and particularly our community, that our software can be trusted. We know it is secured because it has been tested by dozen of hunters and will be continuously tested by new hunters.
```

Comment:
```suggestion
The big aim of having a Bug Bounty Program in public is to show everyone, and particularly our community, that our software can be trusted. We know it is secured because it has been tested by dozen of hunters and will be continuously tested by new hunters.
```